homeassistant/components/alarm_control_panel/__init__.py

```diff
@@ -145,5 +151,6 @@ def state_attributes(self):
         """Return the state attributes."""
         state_attr = {
             ATTR_CODE_FORMAT: self.code_format,
+            ATTR_USER: self.user
```

Comment:
Instead of calling it ATTR_USER, which is very specific, should we call it `changed_by` ? That way it can be a user but it can also be used by other platforms for things like `manual` or maybe even `entity_id` etc